chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,443 @@
+since it will need to be used in subsequent statistical analysis.
+Namely, standard errors for these types of designs must be clustered
+at the level at which the randomization was clustered.\sidenote{
+  \url{https://blogs.worldbank.org/impactevaluations/when-should-you-cluster-standard-errors-new-wisdom-econometrics-oracle}}
+This accounts for the design covariance within the cluster --
+the information that if one individual was observed or treated there,
+the other members of the clustering group were as well.
+
+\subsection{Stratification}
+
+\textbf{Stratification} is a research design component
+that breaks the full set of observations into a number of subgroups
+before performing randomization within each subgroup.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Stratified_Random_Sample}}
+This has the effect of ensuring that members of each subgroup
+are included in all groups of the randomization process,
+since it is possible that a global randomization
+would put all the members of a subgroup into just one of the outcomes.
+In this context, the subgroups are called \textbf{strata}.
+
+Manually implementing stratified randomization in Stata is prone to error.
+In particular, it is difficult to precisely account
+for the interaction of strata sizes with multiple treatment arms.
+Even for a very simple design, the method of randomly ordering the observations
+will often create very skewed assignments.
+This is especially true when a given stratum contains a small number of clusters,
+and when there are a large number of treatment arms,
+since the strata will rarely be exactly divisible by the number of arms.\cite{carril2017dealing}
+The user-written \texttt{randtreat} command properly implements stratification.
+However, the options and outputs (including messages) from the command should be carefully reviewed
+so that you understand exactly what has been implemented.
+Notably, it is extremely hard to target precise numbers of observations
+in stratified designs, because exact allocations are rarely round fractions
+and the process of assigning the leftover ``misfit'' observations
+imposes an additional layer of randomization above the specified division.
+
+Whenever stratification is used for randomization,
+the analysis of differences within the strata (especially treatment effects)
+requires a control in the form of an indicator variable for all strata (fixed effects).
+This accounts for the fact that randomizations were conducted within the strata,
+comparing units to the others within its own strata by correcting for the local mean.
+Stratification is typically used for sampling
+in order to ensure that individuals with relevant characteristics will be observed;
+no adjustments are necessary as long as the sampling proportion is constant across all strata.
+One common pitfall is to vary the sampling or randomization \textit{probability}
+across different strata (such as ``sample/treat all female heads of household'').
```

Comment:
this example is for randomization not sampling.

Reply:
it should apply to both?